test/Generator/ParameterGeneratorTest.php

```diff
@@ -133,8 +133,9 @@ public function testFromReflectionGetDefaultValueNotOptional()
         self::assertCount(2, $params);
 
         $firstParameter = ParameterGenerator::fromReflection($params[0]);
-        self::assertInstanceOf(ValueGenerator::class, $firstParameter->getDefaultValue());
-        self::assertNull($firstParameter->getDefaultValue()->getSourceContent());
+        $valueGenerator = $firstParameter->getDefaultValue();
+        self::assertInstanceOf(ValueGenerator::class, $valueGenerator);
+        self::assertSame('', $valueGenerator->getSourceContent());
```

Comment:
This change looks suspicious

Reply:
Restored BC, now it uses null default value to keep BC.